### ISSUE TITLE
Screen colour picker breaks moving from high PPI screen to low

### ISSUE THREAD

**Issue by dhensby** [ShareX/ShareX#734]
Running setup with Lenovo ThinkPad Carbon X1 (hi resolution screen) and secondary monitor (Dell XPS 1080p via HDMI).

When using colour picker and moving mouse to secondary screen, following error occurs:

```
System.ArgumentOutOfRangeException: Parameter must be positive and < Width.
Parameter name: x
   at System.Drawing.Bitmap.GetPixel(Int32 x, Int32 y)
   at ShareX.ScreenCaptureLib.RectangleRegion.get_CurrentColor()
   at ShareX.ScreenCaptureLib.RectangleRegion.DrawMagnifier(Graphics g)
   at ShareX.ScreenCaptureLib.RectangleRegion.Draw(Graphics g)
   at ShareX.ScreenCaptureLib.Surface.OnPaint(PaintEventArgs e)
   at System.Windows.Forms.Control.PaintWithErrorHandling(PaintEventArgs e, Int16 layer)
   at System.Windows.Forms.Control.WmPaint(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by Jaex** [comment]
I have no idea how two monitor can have different DPI because I don't know multi monitor. Therefore I'm unable to fix this.

**Comment by junderhill** [comment]
I have recreated this issue by moving between my laptop screen (1280*800) and one of my 22" monitors (1920*1080). I'll debug this and have a look at fixing it.

**Comment by junderhill** [comment]
What is interesting is that in the version I have installed, 9.10.1, I get the exception when moving to a monitor of different resolution however in the latest code from the master branch, 9.11 beta, I can't recreate the issue. @dhensby Could you try building the code from master and see if you can recreate the bug?

**Comment by junderhill** [comment]
OK so this issue has been fixed by @Jaex on issue #700 
Relevant commit is here 3e34e5b9d2de9d71d8e3dca282ca5f589fdef462

**Comment by dhensby** [comment]
Ah yes, seems like a duplicate issue and nothing to do with resolution. Thanks
